test.md

```diff
@@ -0,0 +1 @@
+This repo is created with Graphite!
```

Comment:
<i class='graphite__hidden'>[Re: lines 2 to 2]<br/></i>

Cannot see the difference<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/ensorrow/react-diff-view/1?utm_source=unchanged-line-comment">Graphite</a>.</span>